services/mediastoredata/src/it/java/software/amazon/awssdk/services/mediastoredata/TransferEncodingChunkedIntegrationTest.java

```diff
@@ -0,0 +1,201 @@
+    private static final String CONTAINER_NAME = "java-sdk-test-" + Instant.now().toEpochMilli();
+    private static MediaStoreClient mediaStoreClient;
+    private static MediaStoreDataClient syncClientWithApache;
+    private static MediaStoreDataClient syncClientWithUrlConnection;
+    private static MediaStoreDataAsyncClient asyncClientWithNetty;
+    private static AwsCredentialsProvider credentialsProvider;
+    private static Container container;
+    private static PutObjectRequest putObjectRequest;
+    private static DeleteObjectRequest deleteObjectRequest;
+
+    @BeforeAll
+    public static void setup() {
+        credentialsProvider = getCredentialsProvider();
+        mediaStoreClient = MediaStoreClient.builder()
+                                           .credentialsProvider(credentialsProvider)
+                                           .httpClient(ApacheHttpClient.builder().build())
+                                           .build();
+        container = createContainer();
+        URI uri = URI.create(container.endpoint());
+
+        syncClientWithApache = MediaStoreDataClient.builder()
+                                                   .endpointOverride(uri)
+                                                   .credentialsProvider(credentialsProvider)
+                                                   .httpClient(ApacheHttpClient.builder().build())
+                                                   .build();
+
+        syncClientWithUrlConnection= MediaStoreDataClient.builder()
+                                                         .endpointOverride(uri)
+                                                         .credentialsProvider(credentialsProvider)
+                                                         .httpClient(UrlConnectionHttpClient.create())
+                                                         .build();
+
+        asyncClientWithNetty = MediaStoreDataAsyncClient.builder()
+                                                        .endpointOverride(uri)
+                                                        .credentialsProvider(getCredentialsProvider())
+                                                        .httpClient(NettyNioAsyncHttpClient.create())
+                                                        .build();
+
+        putObjectRequest = PutObjectRequest.builder()
+                                           .contentType("application/octet-stream")
+                                           .path("/foo")
+                                           .build();
+
+        deleteObjectRequest = DeleteObjectRequest.builder()
+                                                 .path("/foo")
+                                                 .build();
+    }
+
+    @AfterAll
+    public static void tearDown() throws Exception {
+        syncClientWithApache.deleteObject(deleteObjectRequest);
+        asyncClientWithNetty.deleteObject(deleteObjectRequest);
```

Comment:
Hmm, do we need to delete the same object multiple times here?

Reply:
oops, cut down to just one delete